examples/docs/contracts/Car.md

```diff
@@ -0,0 +1,93 @@
+# Car
+
+## Methods
+
+### move
+
+```solidity
+function move(bool left, bool down, bool right, bool up) external nonpayable returns (bool, bool downPos, bool rightPos, bool upPos)
+```
+
+#### Parameters
+
+| Name  | Type | Description        |
+| ----- | ---- | ------------------ |
+| left  | bool | undefined          |
```

Comment:
That is intended, to have the same behaviour as the public methods which outputs undefined. What would be the + of having empty strings instead of undefined?